db/seeds.rb

```diff
@@ -6,7 +6,31 @@
 FactoryBot.create_list(:language, 20)
 FactoryBot.create_list(:movie, 20)
 FactoryBot.create_list(:person, 20)
-Role.create(name: 'admin', permission_attributes: { can_lock_user: true, can_edit_role: true, can_set_role: true, can_read: true, can_edit: true })
-Role.create(name: 'redactor', permission_attributes: { can_lock_user: false, can_edit_role: false, can_set_role: false, can_read: true, can_edit: true })
-Role.create(name: 'reviewer', permission_attributes: { can_lock_user: false, can_edit_role: false, can_set_role: false, can_read: true, can_edit: false })
-FactoryBot.create_list(:tag, 20)
+admin_role = Role.create(name: 'admin', permission_attributes:
+                        { can_lock_user: true, can_edit_role: true,
+                          can_set_role: true, can_read: true, can_edit: true })
+redactor_role = Role.create(name: 'redactor', permission_attributes:
+                        { can_lock_user: false, can_edit_role: false,
+                          can_set_role: false, can_read: true, can_edit: true })
+reviewer_role = Role.create(name: 'reviewer', permission_attributes:
+                        { can_lock_user: false, can_edit_role: false,
+                          can_set_role: false, can_read: true, can_edit: false })
+
+admin = User.new(email: 'admin@example.com', password: '123456', password_confirmation: '123456',
```

Comment:
User.create(..., confirmed_at: Time.now) instead of 2 lines below. Or am I missing something and skip_confirmation! necessary?